universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java

```diff
@@ -29,6 +31,8 @@ Tag renderInternal(Messages messages, Tag questionFormContent) {
     return renderInternal(messages, questionFormContent, true);
   }
 
+  public abstract ImmutableList<Path> getAllPaths();
```

Comment:
Suggestion (if-minor): what do you think about moving this to services/applicant/question/ApplicantQuestion? I feel putting there is more appropriate given this is not something tied to views

Reply:
Agree

Reply:
Yeah, that's way better even though it was a little bit of code moving.